acceptance/tests/environment/use_enc_environment_for_pluginsync.rb

```diff
@@ -11,30 +11,49 @@
 END
 on master, "chmod 755 #{testdir}/enc.rb"
 
+apply_manifest_on(master, <<-MANIFEST, :catch_failures => true)
+  File {
+    ensure => directory,
+    mode => "0770",
+    owner => #{master.puppet['user']},
+    group => #{master.puppet['group']},
+  }
+  file {
+    '#{testdir}/environments':;
+    '#{testdir}/environments/production':;
+    '#{testdir}/environments/special/':;
+    '#{testdir}/environments/special/modules':;
+    '#{testdir}/environments/special/modules/amod':;
+    '#{testdir}/environments/special/modules/amod/lib':;
+    '#{testdir}/environments/special/modules/amod/lib/puppet':;
+  }
+  file { '#{testdir}/environments/special/modules/amod/lib/puppet/foo.rb':
+    ensure => file,
+    mode => "0640",
+    content => "#special_version",
+  }
+MANIFEST
+
 master_opts = {
+  'main' => {
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.